File: 프로그래머스/1/131705. 삼총사/삼총사.java
class Solution {
    public int solution(int[] number) {
        int ret = 0;
        int size = number.length;
        for(int i = 0; i < size; i++){
            for(int j = i + 1; j < size; j++){
                for(int k = j + 1; k < size; k++){
                    if(number[i] + number[j] + number[k] == 0) ret++;
                }
            }
        }
        return ret;
    }
}

File: 프로그래머스/1/131705. 삼총사/삼총사.py
def solution(number):
    ret = 0
    for i in range(len(number)):
        for j in range(i+1, len(number)):
            for k in range(j + 1, len(number)):
                if number[i] + number[j] + number[k] == 0:
                    ret += 1
    return ret
                    